active_plugins/runstardist.py

```diff
@@ -206,6 +208,25 @@ def create_settings(self):
             maxval=1.0,
             doc=f"""\
 Prevent overlapping 
+""",
+        )
+
+        self.manage_gpu = Binary(
+            text="Manually set how much GPU memory each worker can use?",
+            value=False,
+            doc="""
+If enabled, you can manually set how much of the GPU memory each worker can use.
+This is likely to provide the most benefit on Macs. Do not use in a multi-GPU setup.""",
+        )
+
+        self.manual_GPU_memory_GB = Float(
+            text="GPU memory (in GB) for each worker",
+            value=0.5,
+            minval=0.0000001,
+            maxval=30,
+            doc="""\
+GPU memory in GB available to each worker. Value should be set such that this number times the number
```

Comment:
Can you explicitly explain `minval` and `maxval` and how they relate to `value`?
Do you have a suggestion on how one would determine the value?
Or perhaps even just add in a comment like `We recommend not using manual GPU memory (setting the previous setting to False) if you are not sure what numbers to set here.`

Reply:
Will do but in a separate PR, because easier since this PR is on a fork